com.unity.netcode.gameobjects/Runtime/Simulator/INetworkSimulatorConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Unity.Netcode
+{
+    public interface INetworkSimulatorConfiguration
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public int PacketDelayMs { get; set; }
+        public int PacketJitterMs { get; set; }
+        public int PacketLossInterval { get; set; }
+        public int PacketLossPercent { get; set; }
+        public int PacketDuplicationPercent { get; set; }
```

Comment:
Nit, `public` is unnecessary for interfaces

Reply:
I wonder why I did that. Definitely not necessary, I'll remove it.

Reply:
Ah, I think I remember, I've automatically generated them from the NetworkSimulatorConfiguration class, and the public has just gone unnoticed.